.dockerignore

```diff
@@ -1,4 +1,5 @@
 # More info: https://docs.docker.com/engine/reference/builder/#dockerignore-file
 # Ignore build and test binaries.
 bin/
+!bin/manager
```

Comment:
why do you need to add this entry in `gitignore` ?

Reply:
is on .dockerignore, otherwise the docker/podman will exclude the file when building the image

Reply:
ah yeah my bad